core/shared/src/main/scala/zio/IsSubtype.scala

```diff
@@ -0,0 +1,27 @@
+package zio
+
+import scala.annotation.implicitNotFound
+
+@implicitNotFound("\nOutput Type Mismatch\n  expected: ${B}\n    actual: ${A}\n\n")
```

Comment:
What about something like:

`This operator requires that the output type be a subtype of Option[Int] but the actual type String was not a subtype of Option[Int]`

Reply:
I agree 👍 Wasn't sure about the alignment / terseness. How about just the follow, so as not to repeat the expected type?
`This operator requires that the output type be a subtype of Option[Int] but the actual type was String.`

Reply:
Yes.